### PR TITLE
fixes for pass all

### PR DESCRIPTION
**Pull request by sarp**
This PR contains fixes to pass all for magnet handshake stage

### CODE REVIEW

File: internal/stage_helpers.go [hunk 1]
```diff
@@ -415,7 +416,12 @@ func receiveAndSendHandshake(conn net.Conn, peer PeerConnectionParams) (err erro
 	}
 	
 	if !isEqualToOneOf(handshake.Reserved[:], peer.expectedReservedBytes...) {
-		return fmt.Errorf("did you send reserved bytes? expected bytes: %v but received: %v", peer.expectedReservedBytes[0], handshake.Reserved)
+		var formattedStrings []string
```
Comment (sarp) [inline internal/stage_helpers.go:419]: now shows all possible values for expected bytes

**Comment by rohitpaulk** [comment]
@sarp looks like there's a test failure

**Review by rohitpaulk — APPROVED** [review]
(looks good otherwise!)

**Comment by sarp** [comment]
Not sure why the test is failing because I do have an updated fixture for that test? Any thoughts?

**Comment by sarp** [comment]
is it due to an ordering difference of statements in the output vs fixture?

**Pull request merged**